apps/website-example/pages/_app.tsx

```diff
@@ -144,6 +144,15 @@ function CustomApp({Component, pageProps}: CustomAppProps) {
                 </LogoLink>
               </FooterContainer>
             </Spacer>
+
+            <Script
+              src={publicRuntimeConfig.env.API_URL! + '/static/head.js'}
+              strategy="beforeInteractive"
+            />
+            <Script
+              src={publicRuntimeConfig.env.API_URL! + '/static/body.js'}
```

Comment:
Kannst du das auch noch im website builder als Default einführen? Also das es auch für bajour, tsri babanews und so out of the box funktioniert?